src/main/java/io/appform/functionmetrics/FunctionTimerAspect.java

```diff
@@ -111,26 +111,39 @@ public Object around(ProceedingJoinPoint joinPoint) throws Throwable {
 
         log.trace("Called for class: {} method: {} parameterString: {}", className, methodName, parameterString);
         final FunctionInvocation invocation = new FunctionInvocation(className, methodName, parameterString);
+        final TraceManager traceManager = new TraceManager();
```

Comment:
This will create the trace manager on every invocation of the function.. might not be efficient..

Reply:
Refactored